src/utils/utils.js

```diff
@@ -29,3 +29,11 @@ export const uniqueArray = (arr) => {
   }
   return result
 }
+
+export const cloneNode = (node) => {
+  const dupNode = node.cloneNode(true);
+  ['onclick', 'onmouseover', 'onmouseout'].forEach(function (method) {
```

Comment:
The cloning was already in there, I just extended it.
It'll be there to ensure when the elements are modified in the swal, they don't change any copies outside, or vice-versa.
I guess it's just a sensible thing to do. My changes add the event handler cloning, because cloneNode doesn't clone them be default 😄